### PR TITLE
Fix html syntax

### LINKED ISSUE #261: Invalid html in profiler: <code><pre>
block level element `<pre>` must not be within inline `<code>` element. But the profiler generates such html code in https://github.com/doctrine/DoctrineBundle/blob/master/Resources/views/Collector/db.html.twig#L103

Also it uses invalid `target-data-id` html attribute. Why not use `data-target-id` which would be an allowed data-attribute.

### PR DESCRIPTION
**Pull request by Tobion**
Fixes #261 and some cleanup

### CODE REVIEW

File: Resources/views/Collector/db.html.twig [hunk 1]
```diff
@@ -367,18 +379,4 @@
         }
 
     //]]></script>
-
-    <style>
```
Comment (Tobion) [inline Resources/views/Collector/db.html.twig]: style not allowed in this context

File: Twig/DoctrineExtension.php [hunk 2]
```diff
@@ -38,8 +38,8 @@ public function getFilters()
     {
         return array(
             new \Twig_SimpleFilter('doctrine_minify_query', array($this, 'minifyQuery')),
```
Comment (Tobion) [inline Twig/DoctrineExtension.php]: this doesnt return html

File: Twig/DoctrineExtension.php [hunk 3]
```diff
@@ -229,9 +229,6 @@ public function minifyQuery($query)
             $result = $this->composeMiniQuery($query, $keywords, $required);
         }
 
-        // Remove unneeded boilerplate HTML
-        $result = str_replace(array("<pre style='background:white;'", "</pre>"), array("<span", "</span>"), $result);
```
Comment (Tobion) [inline Twig/DoctrineExtension.php]: leftover useless code

**Comment by stof** [comment]
Thank you @Tobion.

**Pull request closed without merging**